src/app/volunteer/page.tsx

```diff
@@ -1,8 +1,176 @@
+"use client";
+
+declare global {
+  interface Window {
+    submitted: boolean;
+  }
+}
+
+import { useState } from "react";
+import toast from "react-hot-toast";
+import FormHeader from "@/components/home/FormHeader";
+import FormButton from "@/components/FormButton";
+import logo from "@/public/logo.webp";
+import Image from "next/image";
+
 const page = () => {
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  if (isSubmitted) {
+    return (
```

Comment:
You need to add all of this:
![image](https://github.com/user-attachments/assets/534a5275-289a-45bd-a4c1-4ca99681bccd)